dnf.spec

```diff
@@ -168,14 +147,42 @@ Requires:       rpm-plugin-systemd-inhibit
 %else
 Recommends:     (rpm-plugin-systemd-inhibit if systemd)
 %endif
+Provides:       dnf-command(alias)
+Provides:       dnf-command(autoremove)
+Provides:       dnf-command(check-update)
+Provides:       dnf-command(clean)
+Provides:       dnf-command(distro-sync)
+Provides:       dnf-command(downgrade)
+Provides:       dnf-command(group)
+Provides:       dnf-command(history)
+Provides:       dnf-command(info)
+Provides:       dnf-command(install)
+Provides:       dnf-command(list)
+Provides:       dnf-command(makecache)
+Provides:       dnf-command(mark)
+Provides:       dnf-command(provides)
+Provides:       dnf-command(reinstall)
+Provides:       dnf-command(remove)
+Provides:       dnf-command(repolist)
+Provides:       dnf-command(repoquery)
+Provides:       dnf-command(repository-packages)
+Provides:       dnf-command(search)
+Provides:       dnf-command(updateinfo)
+Provides:       dnf-command(upgrade)
+Provides:       dnf-command(upgrade-to)
 
 %description -n python3-%{name}
 Python 3 interface to DNF.
 
 %package automatic
 Summary:        %{pkg_summary} - automated upgrades
 BuildRequires:  systemd
+%if 0%{?fedora} <= 38
```

Comment:
I think we can keep the change to have dnf-automatic depend on python3-dnf rather than dnf in all versions. It does not need /usr/bin/dnf.

Reply:
does it mean that your changes related to dnf automatic apply to all fedoras? If I get it right, I could drop all the added lines resulting in keeping just the following line:
`Requires:       python3-%{name} = %{version}-%{release}`

Reply:
Yep I think it's just like that.

Reply:
In this case I pushed an update

Reply:
Right, thanks!